projects/demo-integrations/cypress/tests/shared.entities.ts

```diff
@@ -0,0 +1,3 @@
+export const EXAMPLE_ID = 'tui-doc-example';
+export const PROPS_AND_METHODS =
+    '%D0%A1%D0%B2%D0%BE%D0%B9%D1%81%D1%82%D0%B2%D0%B0_%D0%B8_%D0%BC%D0%B5%D1%82%D0%BE%D0%B4%D1%8B';
```

Comment:
This looks like Russian tab name from proprietary portal. Should be `API`, I guess

Reply:
Thanks, I delete this one. This is an unnecessary const for deep tests, I'll add it in a future PR